runtime/common/src/impl_moonbeam_xcm_call_tracing.rs

```diff
@@ -31,14 +31,14 @@ macro_rules! impl_moonbeam_xcm_call_tracing {
 				origin: RuntimeOrigin,
 			) -> CallResult {
 				if let Ok(raw_origin) = TryInto::<RawOrigin<AccountId>>::try_into(origin.clone().caller) {
-					match (call.clone(), raw_origin) {
-						(
-							RuntimeCall::EthereumXcm(pallet_ethereum_xcm::Call::transact { xcm_transaction }) |
-							RuntimeCall::EthereumXcm(pallet_ethereum_xcm::Call::transact_through_proxy {
-								xcm_transaction, ..
-							 }),
-							RawOrigin::Signed(account_id)
```

Comment:
Why remove this check?

Reply:
It was not removed, check the limes below